pandas/io/tests/test_clipboard.py

```diff
@@ -80,6 +82,14 @@ def test_round_trip_frame(self):
         for dt in self.data_types:
             self.check_round_trip_frame(dt)
 
+    def test_linesep(self):
+        pd.DataFrame().to_clipboard()
```

Comment:
need a fair amount more tests. these need to exercise all of the paths that are defined above.